### DISCUSSION TITLE
Styling Issues and Custom CSS Implementation

### DISCUSSION THREAD

**Discussion by nirajtrt** [Bitnoise/react-scheduler#176] [Q&A]
@piotrNBTN @mGasiorek998 @adam187 @itlamb @lrbtn 

Hello **react-scheduler** Team,

I hope this message finds you well.

I am currently using **react-scheduler**  in my project and have encountered some challenges related to styling. Specifically, I am looking to implement custom CSS in Tile, but I've found that certain CSS-related props are not available in the library.

I tried to add Styling using auto generated classes but it is changing in Production Mode.

Also, I am unable to add custom data in Tooltip and update Styling for inactive State.

Could you please provide guidance on the best practices for adding custom styles? Additionally, if there are any recommended approaches or workarounds to enhance styling flexibility, I would greatly appreciate your insights.

Thank you for your assistance!

Best regards,
Niraj Narkhede
Github : nirajtrt

![development](https://github.com/user-attachments/assets/4faf3fc8-3665-463d-a264-f98f7d4ee5ac)
![production](https://github.com/user-attachments/assets/854c55b6-5b15-4b21-bfae-f03ca3dee49d)

**Comment by nate-g** [comment]
I've added a PR for customizing tooltips, though I'm not sure how active the maintainers for the project and whether it will be merged soon: https://github.com/Bitnoise/react-scheduler/pull/172

**Comment by nirajtrt** [comment]
Thanks for your response, I will check it out. 

Can you pls add props for **Custom CSS Styling** ? @nate-g
